web-animations-1/Overview.bs

```diff
@@ -4699,6 +4699,10 @@ interface KeyframeEffect : AnimationEffect {
 
     1.  Set the <a>target element</a> of <var>effect</var> to <var>target</var>.
 
+    1.  If <var>options</var> is a {{KeyframeEffectOptions}} object with a
```

Comment:
Is there a reason not to do this in the same step (or at least beside) the setting of `KeyframeEffect/composite` below?

Reply:
done